src/Evolution/Systems/CurvedScalarWave/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@
 set(LIBRARY CurvedScalarWave)
 
 set(LIBRARY_SOURCES
+    Constraints.cpp
```

Comment:
2 spaces here please (I just got asked to make the same change myself in one of my PRs)

Reply:
changed, thanks!